synapse/config/__init__.py

```diff
@@ -12,3 +12,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+from ._base import ConfigError
+
+__all__ = ["ConfigError"]
```

Comment:
Why?

Reply:
because I want to use ConfigError from outside `synapse.config`.

Reply:
You don't need the `__all__` for that?

Reply:
inspired by https://github.com/SublimeLinter/SublimeLinter/issues/167